packages/kit/src/core/config/options.js

```diff
@@ -283,7 +284,8 @@ const options = object(
 );
 
 /**
- * @param {Record<string, Validator>} children
+ * @template {string} Keys
+ * @param {Record<Keys, Validator>} children
```

Comment:
I believe this is left over from the prior changes to validation?

```suggestion
 * @param {Record<string, Validator>} children
```